paddle2onnx/op_mapper/logic.py

```diff
@@ -45,10 +65,38 @@ def opset_1(cls, graph, node, **kw):
 
 @op_mapper('not_equal')
 class NotEqual():
-    support_opset_version_range = (12, )
+    support_opset_version_range = (7, 15)
 
     @classmethod
-    def opset_1(cls, graph, node, **kw):
+    def opset_7(cls, graph, node, **kw):
+        equal_val = None
+        if node.input_dtype('X', 0) in [paddle.float32, paddle.float64]:
+            warning_info = "Equal only support int and bool input, but your input is {}, this may cause wrong result. please try opset version >= 11".format(
```

Comment:
这句英文提示改成下面这样吧
"Operator 'Equal" only support input with dtype of int/bool, now the dtype of input is {}, this may cause wrong results, it is more recommend converting this model with  opset version >= 11."

Reply:
done

Reply:
done